package.json

```diff
@@ -31,28 +31,25 @@
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
     "test:e2e": "jest --config ./test/jest-e2e.json",
     "copyfiles": "mkdir -p dist/generated && cp src/generated/schema.gql dist/generated/schema.gql",
+    "install-ts-client-local": "npm i @nebulr-group/nblocks-ts-client@alpha --registry http://host.docker.internal:4873",
     "npm-publish": "npm run build && npm publish --access public",
     "npm-publish-beta": "npm run build && npm publish --access public --tag beta",
-    "npm-publish-local": "npm run build && npm publish --registry http://verdaccio:4873 --tag alpha"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm run format && npm run lint"
-    }
+    "npm-publish-local": "npm run build && npm publish --registry http://host.docker.internal:4873 --tag alpha",
+    "build:watch": "concurrently \"tsc --watch\" \"chokidar 'src/generated/schema.gql' -c 'npm run copyfiles'\""
   },
   "dependencies": {
-    "@nebulr-group/nblocks-ts-client": "^5.1.0",
-    "@nestjs/common": "^7.0.0 || ^8.1.2",
-    "@nestjs/config": "^0.5.0 || ^1.0.2",
-    "@nestjs/core": "^7.0.0 || ^8.1.2",
-    "@nestjs/graphql": "^7.6.0 || ^8.0.2",
-    "@nestjs/mongoose": "^7.2.0 || ^8.0.1",
+    "@nebulr-group/nblocks-ts-client": "^5.4.2",
+    "@nestjs/common": "^8.4.7",
+    "@nestjs/config": "^1.0.2",
+    "@nestjs/core": "^8.4.7",
+    "@nestjs/graphql": "^8.0.2",
+    "@nestjs/mongoose": "^9.2.0",
```

Comment:
This will most certainly have implications on apps that use this plugin. e.g. Northwhistle.